docs/src/getting-started/system-dependencies.md

```diff
@@ -0,0 +1,52 @@
+# System Requirements
+
+There are several system requirements including forc, llvm, clang and postgres.
+
+## Fuel system dependencies
+
+Getting started with a Fuel indexer requires a few system dependencies from the Fuel ecosystem, namely, `forc`, and `fuel-core`
+- `forc` is the crate that holds the Sway language and Fuel's equivalent of `cargo`
+- `fuel-core` is the crate that contains the Fuel node software and execution
+
+### Installation
+
+On all architectures, install the following Fuel crates via `cargo`
+
+```bash
+cargo install forc fuel-core
+```
+
+## Other system dependencies
+
```

Comment:
It might be worth mentioning generally what the main dependencies are and why they're needed before going on to how to install them. It could help us to keep track too in case we'd like to try and simplify the installation process in the future.

It looks like the main ones are:

- **cmake** + **clang** + **pkg-config** - I guess these must be for building the bindings for one of our crate deps?
- **OpenSSL** - We might not need this anymore. I think `forc` was updated a little while ago to link it statically. That said, there might be another dep that needs it?
- **postgresql** - DB stuff.

All that said, we can add something like this in a future PR if you'd prefer to land the rest first :+1:

Reply:
@mitchmindtree 
- Agreed, it would be better to clarify why these deps are needed. 
- https://github.com/FuelLabs/fuel-indexer/issues/28 was created to address this. 
  - I'd vote to merge as is, and put a deadline on #28 (I'll actually tackle that while I'm doing this doc work)